File: runtime/Jlib/rest.java
package bigloo;

public final class rest extends cnst
{
  public static final rest rest= new rest( 0x103 );

  private rest( final int  n )
  {
    super( n );
  }

  public void write( final output_port  p )
  {
    p.write( "#!rest" );
  }
}

File: runtime/CSlib/rest.cs
namespace bigloo
{
  public sealed class rest: cnst
  {
    public static readonly rest _rest= new rest( 0x103 );

    private rest( int  n )
      : base( n ) 
    {
    }

    public override void write( output_port  p )
    {
      p.write( "#!rest" );
    }
  }
}
